irohad/network/impl/async_grpc_client.hpp

```diff
@@ -75,6 +75,17 @@ namespace iroha {
         std::unique_ptr<grpc::ClientAsyncResponseReader<Response>>
             response_reader;
       };
+
+      /**
+       * Universal method to perform all needed sends
+       * @tparam lambda is lambda which must return unique pointer to ClientAsyncResponseReader<Response> object
```

Comment:
`clang-format`

Reply:
Also, there's not need to write `lambda is lambda which`, can be just `lambda which must...`